spec/helpers/pages_helper_spec.rb

```diff
@@ -11,7 +11,7 @@
     before do
       allow(helper).to receive(:signed_in?).with(no_args).and_return(signed_in?)
       allow(helper).to receive(:render).with(
-        partial: 'product_button',
+        partial: 'shared/product_button',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.